lib/dev/ui/spinner.rb

```diff
@@ -31,7 +31,7 @@ class Task
           def initialize(title, &block)
             @title = title
             @thread = Thread.new do
-              cap = Dev::UI::StdoutRouter::Capture.new(with_frame_inset: false, &block)
+              cap = Dev::UI::StdoutRouter::Capture.new(self, with_frame_inset: false, &block)
```

Comment:
As far as what we yield to the block goes, eventually when there is more stuff, we probably want to yield a proxy/control object, but I think for now what I did is the simplest option